modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -8,6 +8,8 @@
 
 The `wasp-agent` component facilitates memory overcommitment by assigning swap resources to worker nodes. It also manages pod evictions when nodes are at risk due to high swap I/O traffic or high utilization.
 
+In new, high density deployments where the `memoryOvercommitPercentage` in the `HyperConverged` CR is initially set to 100 or more, the `wasp-agent` component is deployed automatically. 
```

Comment:
```suggestion
In new, high-density deployments where the `memoryOvercommitPercentage` in the `HyperConverged` CR is initially set to `100` or more, the `wasp-agent` component is deployed automatically. 
```

Tiny nits: hyphen; backticks for the value.

The word "initially" is also giving me pause. Does it mean "when the `HyperConverged` CR was first created"?

If so, I might rephrase this to be more specific. Suggestion:
```
The `wasp-agent` component is deployed automatically if `memoryOvercommitPercentage` is set to `100` or more when you first create the `HyperConverged` custom resource (CR).
```